Tests/StreamChatTests/Database/DatabaseContainer_Tests.swift

```diff
@@ -365,4 +330,84 @@ final class DatabaseContainer_Tests: XCTestCase {
             XCTAssertEqual(database.backgroundReadOnlyContext.shouldShowShadowedMessages, shouldShowShadowedMessages)
         }
     }
+    
+    // MARK: -
+    
+    private func writeALotOfData(to container: DatabaseContainer) throws {
```

Comment:
Maybe this name is more clear, because it is not just a lot of data, it needs to be all entities
```suggestion
    private func writeDataForAllEntities(to container: DatabaseContainer) throws {
```